platforms/ios/build_framework.py

```diff
@@ -70,17 +79,32 @@ def _build(self, outdir):
 
         xcode_ver = getXCodeMajor()
 
-        for t in self.targets:
+        if self.dynamic:
+            alltargets = self.targets
+        else:
+            # if we are building a static library, we must build each architecture separately
+            alltargets = []
+
+            for t in self.targets:
+                for at in t[0]:
+                    current = ( [at], t[1] )
+
+                    alltargets.append(current)
+
+        for t in alltargets:
             mainBD = self.getBD(mainWD, t)
             dirs.append(mainBD)
+
             cmake_flags = []
             if self.contrib:
                 cmake_flags.append("-DOPENCV_EXTRA_MODULES_PATH=%s" % self.contrib)
-            if xcode_ver >= 7 and t[1] == 'iPhoneOS':
+            if xcode_ver >= 7 and t[1] == 'iPhoneOS' and self.bitcode == False:
                 cmake_flags.append("-DCMAKE_C_FLAGS=-fembed-bitcode")
                 cmake_flags.append("-DCMAKE_CXX_FLAGS=-fembed-bitcode")
```

Comment:
Is it correct: "and self.bitcode == False"? Because this code adds bitcode generation parameters.